examples/godot_tps_controller_port/.gitignore

```diff
@@ -0,0 +1,2 @@
+# Player assets are acquired manually
+/Player
```

Comment:
Sorry. One more thing: are you sure that this is the correct case? I just downloaded a fresh copy of the original project, and the directory name was `player`, not `Player`. Yes, there are many file systems (actually, about everything outside Windows) that care about case.

If the capitalized version is actually required for Windows, add both to `.gitignore`. Otherwise, replace it with `player`.